gap_analysis_project/README.rst

```diff
@@ -0,0 +1,79 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+{gap_analysis_project}
+==============
+
+linking business requirements to project.
```

Comment:
This module is part of a set ("Gap Analysis") and provides the possibility to
create tasks in a project based on the business analysis estimation content.